sales_team_multicompany/sales_team_multicompany_demo.xml

```diff
@@ -0,0 +1,14 @@
+<?xml version="1.0" encoding="utf-8"?>
+<odoo>
+    <data noupdate="1">
```

Comment:
Is this data originally `noupdate="1"`. If so, it won't be updated with this definition. You need to set a pre and post-init-hook to remove the noupdate flag and after the installation, to restore it.

You can find an example here: https://github.com/OCA/l10n-spain/pull/214/files#diff-d1d1fa00d15ff44bb400415153c94f00R30 (but you have to use latest conventions that move these methods to the file hooks.py).